.github/workflows/linux-build.yml

```
@@ -17,42 +1,16 @@
-concurrency:
-  group: ${{ github.workflow }}-${{ github.repository }}-${{ github.head_ref || github.sha }}
-  cancel-in-progress: true
+      - run: "pwd"
+        shell: "bash"
 
-jobs:
-  linux-gcc:
-    name: Build with GCC
-    uses: ./.github/workflows/linux-build-base.yml
+      - run: "echo $GITHUB_WORKSPACE"
+        shell: "bash"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)